### PR TITLE
publish deno_fetch during CI

### PR DESCRIPTION
**Pull request by ry**

### CODE REVIEW

File: op_crates/fetch/lib.rs [hunk 1]
```diff
@@ -245,21 +236,13 @@ where
 /// Create new instance of async reqwest::Client. This client supports
 /// proxies and doesn't follow redirects.
 fn create_http_client(ca_file: Option<&str>) -> Result<Client, AnyError> {
-  let mut headers = HeaderMap::new();
-  // TODO(ry) set the verison correctly.
-  headers.insert(USER_AGENT, format!("Deno/{}", "x.x.x").parse().unwrap());
-  let mut builder = Client::builder()
-    .redirect(Policy::none())
-    .default_headers(headers)
-    .use_rustls_tls();
```
Comment (bartlomieju) [inline op_crates/fetch/lib.rs:254]: We should pass version explicitly instead of removing

**Comment by piscisaureus** [comment]
Lots of warnings
Easiest is probably: `cargo default nightly; cargo fix --allow-dirty --workspace`

**Comment by ry** [comment]
Those warnings aren't introduced in this PR

**Review by bartlomieju — APPROVED** [review]
LGTM

**Pull request merged**